homeassistant/components/bangolufsen/manifest.json

```
@@ -0,0 +1,11 @@
+{
+  "domain": "bangolufsen",
```

Wouldn't bang_olufsen be a better fit?
```suggestion
  "domain": "bang_olufsen",
```

---

This is simply a styling choice that we prefer, but it is not a dealbreaker if the latter is more "Home Assistant" like.

Sorry for the late response, I wanted to take time properly fix the code. I want to say thank you for the thorough code review. It really means a lot!

Kind regards

---

`bang_olufsen` would indeed be more "Home Assistant" like, please change it 👍

---

I've changed the name to bang_olufsen.